.travis.yml

```diff
@@ -6,13 +6,14 @@ python:
     - "2.7"
 
 install:
-    - pip install coveralls flake8
+    - pip install coveralls flake8 unittest2
```

Comment:
Why do you need `unittest2`? Maybe I'm wrong, but the result will be the same if your test class extends `unittest.TestCase` in place of `unittest2.TestCase`